### DISCUSSION TITLE
libfp GUI/Frontend Ideas and Discussion

### DISCUSSION THREAD

**Discussion by SecularSteve** [oblivioncth/CLIFp#97] [General]
https://github.com/oblivioncth/libfp

Someone's attempting to create a C++ based GUI/Frontend for both CLIFp and FIL.

It has potential to be a big-screen like mode to launch Flash games natively on platforms where it's not fully implemented, like Linux or Mac.

Though it hasn't been touched in months... Seems like a good thing to result from this as a base.

It's currently incomplete, and not really ready for use. But so far from what I see, it could be the best solution.

**Comment by oblivioncth** [comment]
Sorry for the radio silence, been dealing with some stuff.

I'm... confused. If you're referring to the project at the link you provided, that's another one of my repositories lol. I apologize if I'm misunderstanding something, but I think you may have taken "interface" in that repo's description to mean "frontend", but it's actually quite the opposite. That repo is a C++ library which acts as an API (hence 'interface') so that other applications can interact with Flashpoint without having to do a lot of legwork themselves. So it is actually the foundation that CLIFp and FIL are built-on, not the other way around.

**Reply by SecularSteve** [reply]
Wah wah waaaaahhhh 🎺

I dunno why I didn't see that, I feel like a jackass 🐴

I apologise for that.
